src/components/userGroups/User.tsx

```diff
@@ -375,109 +377,119 @@ export default function UserForm({
-                                ...base,
-                                border: `1px solid var(--N200)`,
-                                borderRadius: `4px`,
-                                background: 'white',
-                                height: '30px',
-                                margin: '0 8px 0 0',
-                                padding: '1px',
-                            }),
-                        }}
-                        formatOptionLabel={formatChartGroupOptionLabel}
-                        closeMenuOnSelect={false}
-                        isMulti
-                        autoFocus={!id}
-                        name="groups"
-                        options={availableGroups}
-                        hideSelectedOptions={false}
-                        onChange={(selected, actionMeta) => setUserGroups((selected || []) as any)}
-                        className={`basic-multi-select ${id ? 'mt-8 mb-16' : ''}`}
-                    />
-                    <AppPermissions
-                        data={userData}
-                        directPermission={directPermission}
-                        setDirectPermission={setDirectPermission}
-                        chartPermission={chartPermission}
-                        setChartPermission={setChartPermission}
-                        k8sPermission={k8sPermission}
-                        setK8sPermission={setK8sPermission}
-                        currentK8sPermissionRef={currentK8sPermissionRef}
-                    />
+                    <div className="flex left mb-16">
+                        <RadioGroup
+                            className="permission-type__radio-group"
+                            value={localSuperAdmin}
+                            name={`permission-type_${id}`}
+                            onChange={handlePermissionType}
+                        >
+                            {PermissionType.map(({ label, value }) => (
+                                <RadioGroupItem
+                                    dataTestId={`${
+                                        value === 'SPECIFIC' ? 'specific-user' : 'super-admin'
+                                    }-permission-radio-button`}
+                                    value={value}
+                                    key={label}
+                                >
+                                    <span
+                                        className={`dc__no-text-transform ${
+                                            localSuperAdmin === value ? 'fw-6' : 'fw-4'
+                                        }`}
+                                    >
+                                        {label}
+                                    </span>
+                                </RadioGroupItem>
+                            ))}
+                        </RadioGroup>
+                    </div>
+                    {localSuperAdmin === 'SPECIFIC' && (
```

Comment:
make it enum

Reply:
will handle in the active/inactive story